### PR TITLE
RadionButton control for WPF

### PR DESCRIPTION
**Pull request by yurkinh**
### Description of Change ###
Added RadioButton Renderer for WPF

### Issues Resolved ### 
Missing RadioButton control for WPF

### Platforms Affected ### 

- WPF

### Before/After Screenshots ### 
Radiobuttongif

### Testing Procedure ###
Control Gallery

### PR Checklist ###

- [x] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.Platform.WPF/FormsRadioButton.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+		void UpdateBackgroundColor()
+		{
+			if (BackgroundColor == null)
+				BackgroundColor = Background;
+
+			Background = BackgroundColor;
+		}
+
+		void UpdateBorderRadius()
```
Comment (jsuarezruiz) [inline Xamarin.Forms.Platform.WPF/FormsRadioButton.cs:67]: This empty method does not seem necessary, we can eliminate it.

File: Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+using WThickness = System.Windows.Thickness;
+
+namespace Xamarin.Forms.Platform.WPF
+{
+	public class RadioButtonRenderer : ViewRenderer<RadioButton, FormsRadioButton>
+	{
+		bool _fontApplied;
+		FormsRadioButton _button;
+
+		public bool IsDisposed { get; private set; }
```
Comment (jsuarezruiz) [inline Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs:13]: Is a public property necessary here? If it is not, change it to use a bool variable bool `_isDisposed`.

File: Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+			if (e.NewElement != null)
+			{
+				if (Control == null)
+				{
+					_button = new FormsRadioButton();
+
+					_button.Click += OnButtonClick;
+					_button.AddHandler(System.Windows.Controls.Button.ClickEvent, (RoutedEventHandler)OnPointerPressed, true);
+					_button.Loaded += ButtonOnLoaded;
```
Comment (jsuarezruiz) [inline Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs:27]: `OnButtonLoaded`

File: Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+				if (Element.IsSet(RadioButton.PaddingProperty) && Element.Padding != (Thickness)RadioButton.PaddingProperty.DefaultValue)
+					UpdatePadding();
+
+				UpdateFont();
+				UpdateCheck();
+			}
+		}
+
+		void ButtonOnLoaded(object o, RoutedEventArgs routedEventArgs)
```
Comment (jsuarezruiz) [inline Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs:60]: Don't access anything related to the native control when loaded. If not necessary, remove the method.

File: Xamarin.Forms.Platform.WPF/FormsRadioButton.cs (hunk 1 above)
Comment (yurkinh) [reply Xamarin.Forms.Platform.WPF/FormsRadioButton.cs:67]: Done

File: Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs (hunk 2 above)
Comment (yurkinh) [reply Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs]: Done

File: Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs (hunk 3 above)
Comment (yurkinh) [reply Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs]: Done

File: Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs (hunk 4 above)
Comment (yurkinh) [reply Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs]: Done

**Pull request merged**